website/docs/add-secure-apps/flows-stages/stages/redirect/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Redirect stage
+---
+
+This stage's main purpose is to redirect the user to a new Flow while keeping flow context. For convenience, it can also redirect the user to a static URL.
+
+## Redirect stage modes
+
+### Static mode
+
+When the user reaches this stage, they are redirected to a static URL.
+
+### Flow mode
+
+When the user reaches this stage, they are redirected to a Flow, retaining all [flow context](../../flow/context).
+
+Optionally, untoggle the "Keep flow context" switch. If this is untoggled, all flow context is cleared with the exception of the [is_redirected](../../flow/context#is_redirected-flow-object) key.
```

Comment:
```suggestion
Optionally, untoggle the "Keep flow context" switch. If this is untoggled, all flow context is cleared with the exception of the [`is_redirected`](../../flow/context#is_redirected-flow-object) key.
```

Reply:
I think this is a bit worse, because the link no longer looks like a link (to me).

![image](https://github.com/user-attachments/assets/18841c12-3a97-4838-882b-9ae1d4cf945d)

vs

![image](https://github.com/user-attachments/assets/7ebfa1e6-f158-4b10-b97b-8a4a0fe1c5df)

Let me know if you'd like it changed anyway, I'm not categorically opposed.